### ISSUE TITLE
Error for LowLevelChecks not very helpful

### ISSUE THREAD

**Issue by omgmog** [bolt/bolt#2995]
```
PHP Fatal Error: Bolt Core

Error: Uncaught exception 'Bolt\Configuration\LowlevelException' in src/Configuration/LowlevelChecks.php:189
Stack trace:
#0 src/Configuration/ResourceManager.php(344): Bolt\Configuration\LowlevelChecks->doDatabaseCheck()
#1 src/Configuration/ResourceManager.php(284): Bolt\Configuration\ResourceManager->verifyDb()
#2 src/Configuration/ResourceManager.php(275): Bolt\Configuration\ResourceManager->postInitialize()
#3 src/Application.php(48): Bolt\Configuration\ResourceManager->initialize()
#4 app/bootstrap.php(79): Bolt\Application->__construct(Array)
#5 [internal function]: Bolt\{closure}()
#6 app/bootstrap.php(86): call_user_func(Object(Closure))
#7 app/load.php(40): require_once('a...')
#8 index.php(13): require_once('a...')
#9 {main}
thrown
File: src/Configuration/LowlevelChecks.php
Line: 189
```

This issue is caused by not having `pdo_sqlite` available, but you can only work that out by going to `src/Configuration/LowlevelChecks.php` and seeing that the problem is `SQLite was selected as the database type, but the driver does not exist or is not loaded. Please install the pdo_sqlite driver.`

**Comment by GawainLynch** [comment]
Thank @omgmog! 

This is indeed undesirable behaviour, and has been fixed in the upcoming v2.1 that went into beta this morning.

Now you will be shown the following (just tested it myself on a fresh beta install):
```
Bolt - Fatal Error
SQLite was selected as the database type, but the driver does not exist or is not loaded. 
Please install the pdo_sqlite driver.

This is a fatal error. Please fix the error, and refresh the page. Bolt can not run, until 
this error has been corrected. 
Make sure you've read the instructions in the documentation for help. If you can't get 
it to work, post a message on our forum, and we'll try to help you out. Be sure to 
include the exact error message you're getting!

Bolt documentation - Setup
Bolt questions on Stack Overflow
```

**Comment by omgmog** [comment]
Good news, thanks!
